### ISSUE TITLE
Plans for Contao 4

### ISSUE THREAD

**Issue by Aybee** [netzmacht/contao-timelinejs#5]
Hello, are you planning a version for Contao 4?

**Comment by dmolineus** [comment]
Yes, there is already a version for Contao 4. It's not published at open source at the moment because the development costs aren't covered yet.

You can find more information in the contao community:
https://community.contao.org/de/showthread.php?43696-timelinejs-Zeitstrahl-in-Contao-verwalten/page2&p=487639&viewfull=1#post487639
